build.assets/Dockerfile

```diff
@@ -50,13 +50,18 @@ RUN (mkdir -p /opt && cd /opt && curl https://storage.googleapis.com/golang/go1.
 RUN groupadd jenkins --gid=$GID -o && useradd jenkins --uid=$UID --gid=$GID --create-home --shell=/bin/sh
 RUN (mkdir -p /var/lib/teleport && chown -R jenkins /var/lib/teleport)
 
+# get etcd
+RUN (curl -L https://github.com/coreos/etcd/releases/download/v3.0.16/etcd-v3.0.16-linux-amd64.tar.gz | tar -xz ;\
```

Comment:
This will make everything more complex and convoluted. $UID and $GID are used because they are _different from machine to machine_. The version of etcd is a constant (just like the version of Go) and is set by changing a single line in Dockerfile just like any other software.